tests/dbgp/dbgpclient.php

```diff
@@ -1,16 +1,53 @@
 <?php
 class DebugClient
 {
+	protected $triedPorts = 0;
+
+	protected $startPort = 9991;
+
 	private $tmpDir;
 
+	protected function getPort()
+	{
+		return $this->startPort + $this->triedPorts;
+	}
+
+	protected function getIPAddress()
+	{
+		$records = dns_get_record(php_uname('n'), DNS_A);
+
+		if(!empty($records))
```

Comment:
I have fixed this on my local file. Feels like a strange way of writing code when I'm used to other coding standards. But of course I want to follow the correct one for this project.